src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: OTEL log ingestion solution for ADOT lambda layer collector
+metaDescription: How to instrument your serverless applications on AWS Lambda with ADOT solution in New Relic.
+freshnessValidatedDate: never
+---
+You can now monitor AWS serverless application that sends OTEL log events data from CloudWatch Logs to the New Relic UI.
+You can configure the ADOT lambda layer collector to successfully forward OTEL application logs with all necessary service and application context. 
+This includes ensuring that logs contain metadata such as service name, version, environment, and any other relevant attributes that can aid in observability and troubleshooting.
+
+
+## Pre-requisites
```

Comment:
Your headers need anchor tags: https://docs.newrelic.com/docs/style-guide/structure/levels-headings/#h2_identify

Reply:
Also, I'm pretty sure it's "Prerequisites"